test/test_from_sdist.py

```diff
@@ -61,10 +61,23 @@ def test_simple(tmp_path):
     sdist_dir.mkdir()
     sdist_path = make_sdist(basic_project, sdist_dir)
 
+    setup_py_assertion_snippet = textwrap.dedent(
+        """
+        import os
+
+        assert os.path.exists("setup.py")
+        assert os.path.exists("{package}/setup.py")
+        """,
+    )
+    setup_py_assertion_cmd = f"python3 -c '{setup_py_assertion_snippet !s}'"
```

Comment:
I think Windows requires the CLI arguments to be wrapped with double quotes and breaks on single quotes. This should work for both Windows and *nix:
```suggestion
    setup_py_assertion_cmd = f'python3 -c "{setup_py_assertion_snippet !s}"'
```